### DISCUSSION TITLE
CreateContainerError due to docker-engine runtime

### DISCUSSION THREAD

**Discussion by digitalnomd** [moby/moby#44951] [General]
What would be the possible causes of `CreateContainerError` within the docker-engine runtime? I'm experiencing this on docker-engine v20.10.6.

**Comment by thaJeztah** [comment]
Do you have more context and steps to reproduce? `CreateContainerError` does not look to be an error the engine itself produces, but I see it being generated by other projects; https://grep.app/search?q=CreateContainerError
